docs/common/yum.md

```diff
@@ -1,167 +1,108 @@
-
-YUM Repositories
-================
+**YUM Repositories**
```

Comment:
It doesn't look like the bolded header is any different. Can we just make this the level 1 heading and bump the others down to level 2?